src/utils.js

```diff
@@ -13,6 +13,26 @@ import { PACKAGE_TYPES, LOCAL_PROTOCOLS } from 'const';
 
 const SOURCE_MAP_RE = new RegExp(/\/\/[#@]\s(source(?:Mapping)?URL)=\s*(\S+)/);
 
+export class AddonsLinterUserError extends Error {
```

Comment:
Maybe add some explanations about this special error.

Reply:
+1 In 661f1cf I added an inline comment right above the class to provide some details about when this error class is supposed to be used and what kind of special handling is being done on this kind of errors.